landingzones/tests/test_permissions_taskflow.py

```diff
@@ -0,0 +1,222 @@
+"""Tests for UI view permissions with taskflow"""
+
+from django.test import override_settings
+from django.urls import reverse
+
+# Projectroles dependency
+from projectroles.models import SODAR_CONSTANTS
+from projectroles.tests.test_permissions import TestPermissionMixin
+
+# Samplesheets dependency
+from samplesheets.tests.test_io import SampleSheetIOMixin, SHEET_DIR
+
+from taskflowbackend.tests.base import (
+    TaskflowAPIPermissionTestBase,
+    TaskflowTestMixin,
+)
+from landingzones.tests.test_models import (
+    LandingZoneMixin,
+    ZONE_TITLE,
+    ZONE_DESC,
+)
+from landingzones.tests.test_views_taskflow import LandingZoneTaskflowMixin
+
+
+# SODAR constants
+PROJECT_ROLE_OWNER = SODAR_CONSTANTS['PROJECT_ROLE_OWNER']
+PROJECT_ROLE_DELEGATE = SODAR_CONSTANTS['PROJECT_ROLE_DELEGATE']
+PROJECT_ROLE_CONTRIBUTOR = SODAR_CONSTANTS['PROJECT_ROLE_CONTRIBUTOR']
+PROJECT_ROLE_GUEST = SODAR_CONSTANTS['PROJECT_ROLE_GUEST']
+PROJECT_TYPE_CATEGORY = SODAR_CONSTANTS['PROJECT_TYPE_CATEGORY']
+PROJECT_TYPE_PROJECT = SODAR_CONSTANTS['PROJECT_TYPE_PROJECT']
+
+# Local constants
+SHEET_PATH = SHEET_DIR + 'i_small.zip'
+TEST_OBJ_NAME = 'test1.txt'
+
+
+class TestLandingZonePermissionTaskflowBase(
+    LandingZoneMixin,
+    LandingZoneTaskflowMixin,
+    SampleSheetIOMixin,
+    TestPermissionMixin,
+    TaskflowTestMixin,
+    TaskflowAPIPermissionTestBase,
```

Comment:
This should be based on an UI taskflow permission test base class, but it looks like one doesn't currently exist. I'll add a separate ticket for this. For now, if this works, feel free to keep it as is.